.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. If there is a duplicate, please close your issue and add a comment
+to the existing issue.
+
+If you suspect your issue is a bug, please edit your issue description to
+include the information shown below.
+
+For more information about reporting issues, see
+https://github.com/openwhisk/openwhisk/blob/master/CONTRIBUTING.md#raising-issues
+
+
+Use the commands below to provide key information from your environment:
+You do NOT have to include this information if this is a feature request
+-->
+
+**Environment details (Operating System, ...):**
+
+
+**Steps to reproduce the issue:**
+1.
+2.
+3.
+
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Additional information you deem important (e.g. issue happens only occasionally):**
```

Comment:
#### Additional information you deem important
e.g. issue happens only occasionally or under certain circumstances
e.g. changes you did or observed in the environment